tests/test_graph_api.py

```diff
@@ -684,6 +685,7 @@ def test_get_unicode_url():
     mock_request.return_value.status_code = 200
 
     response = graph.get('https://www.facebook.com/christophernewportuniversity‎')
+    del response['headers']
```

Comment:
yeah I thought about that but `response['headers']` is actually an instance of some mock request something or other.  TBH I've never used `mock` before. I'll take a closer look and give it a shot.